.github/workflows/codeql.yml

```diff
@@ -0,0 +1,46 @@
+name: "CodeQL"
```

Comment:
We use `codeql-analysis.yml` on other repositories. I don't have a strong preference as to which to use, but should standardize, imho.

Reply:
Which standard would be easier to adopt?  GitHub choose the `codeql.yml` name as a default

Reply:
I only see three cases of `codeql-analysis.yml`.  If it sounds good I suggest we transition to the default naming to reduce the manual steps of on-boarding a project.

I can submit PR's for the 3 projects we have today